### ISSUE TITLE
Some verbatim text not showing in POD

### ISSUE THREAD

**Issue by book** [metacpan/metacpan-web#1228]
The syntax highlighter that has a few issues with the verbatim text in the [perlsecret](https://metacpan.org/pod/distribution/perlsecret/) manual page:
- https://metacpan.org/pod/distribution/perlsecret/lib/perlsecret.pod#Bang-bang should show `!!` but doesn't
- https://metacpan.org/pod/distribution/perlsecret/lib/perlsecret.pod#Eskimo-greeting should show `}{` but doesn't
- https://metacpan.org/pod/distribution/perlsecret/lib/perlsecret.pod#Inchworm should show `~~` but doesn't

When viewing the source with my browser, the HTML code looks fine:

```
<h2 id="Bang-bang">Bang bang</h2>
<pre class="brush: pl; class-name: 'highlight'; toolbar: false; gutter: false; metacpan-verbatim">    !!</pre>
```

but when looking with the browser's inspector, the DOM looks like this:

```
<h2 id="Bang-bang">Bang bang</h2>
 <div>
  <div id="highlighter_440921" class="syntaxhighlighter nogutter highlight pl">
   <table border="0" cellpadding="0" cellspacing="0">
    <tbody>
     <tr>
      <td class="code">
       <div class="container">
        <div class="line number1 index0 alt2">&nbsp;</div>
       </div>
      </td>
     </tr>
    </tbody>
   </table>
  </div>
 </div>
```

which makes me think that a JavaScript syntax highlighter is used, and when it doesn't understand the given text, it just outputs `&nbsp;`. In my opinion, it should just return non-highlighted text.

**Comment by Smylers** [comment]
Another example: the code block of just `1;` is invisible at: https://metacpan.org/pod/Moops#Outer-Sugar

HTML source:

````
<pre><code>   1;
   </code></pre>
````

Generated dom — slightly different from @book's above, because it still contains the `<pre>` element:
````
<pre>
  <div>
      <div id="highlighter_437710" class="syntaxhighlighter nogutter  perl">
      <table border="0" cellpadding="0" cellspacing="0">
        <tbody>
         <tr>
            <td class="code">
              <div class="container">
                <div class="line number1 index0 alt2">&nbsp;</div>
              </div>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</pre>
````

**Comment by Grinnz** [comment]
Another example https://metacpan.org/pod/distribution/RL/bin/pl#SYNOPSIS

This is clearly caused by the javascript as the contents are visible with javascript disabled.

**Comment by shlomif** [comment]
I was told this is the same bug - https://github.com/syntaxhighlighter/syntaxhighlighter/issues/15 - it was closed.

**Comment by matthewpersico** [comment]
Mentioned again in https://www.reddit.com/r/perl/comments/bh9etr/no_matter_what_well_always_have_more_fun_than_the/

**Comment by oalders** [comment]
Closed via #2292.  Thanks @gordon-fish for the fix and everyone else for the comments.  This bug was squashed before its 6th birthday!
